src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFile.java

```diff
@@ -108,8 +110,10 @@ public String getDisplayName() {
 
         public ListBoxModel doFillFileIdItems(@AncestorInPath ItemGroup context, @AncestorInPath Item project) {
             // You should have permission to configure your project in order to get the available managed files
-            if (project != null) {
+            if (context != null) {
                 project.checkPermission(Item.CONFIGURE);
+            } else {
+                Jenkins.get().checkPermission(Item.CREATE);
```

Comment:
not convinced this is correct.  can you have an `ItemGroup` in Jenkins that is not an `Item` (which would lead to checking the wrong thing.

```suggestion
                context.checkPermission(Item.CREATE);
```